### PR TITLE
18 new step print message

### PR DESCRIPTION
**Pull request by nils-wernecke**
New step which prints messages into the Terminal and Report.

`*Print message <Message>`

### CODE REVIEW

**Review by mario-s — APPROVED** [review]

**Review by derdualist — COMMENTED** [review]
Kannst Du den Step noch in der docs/STEPS.md dokumentieren?

File: docs/STEPS.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ The following Gauge steps are implemented in this module:
   - [Assert page screenshots resemble \<file> with SSIM more than \<threshold>](#assert-page-screenshots-resemble-file-with-ssim-more-than-threshold)
   - [Assert page screenshots resemble \<file> with SSIM more than \<threshold> for \<pages>](#assert-page-screenshots-resemble-file-with-ssim-more-than-threshold-for-pages)
   - [Fail \<message>](#fail-message)
+  - [Print message \<message>](#print-message-message)
```
Comment (mario-s) [inline docs/STEPS.md:97]: Wait, the documentation is missing.

**Review by mario-s — CHANGES_REQUESTED** [review]
Add some documentation please.

File: docs/STEPS.md [hunk 2]
```diff
@@ -1781,3 +1782,15 @@ Support
 |Desktop|Android (Chrome)|iOS (Safari)|
 |:-----:|:--------------:|:----------:|
 |   ✔   |       ✔        |     ✔      |
+
+## Print message \<message>
+
+> \* Print message "Please finish this scenario"
+
+This step will leave the specified message in the report and print it in the terminal.
```
Comment (mario-s) [inline docs/STEPS.md:1790]:
```suggestion
This step adds the specified message in the report and print it in the terminal.
```

**Review by mario-s — APPROVED** [review]

File: docs/STEPS.md [hunk 3]
```diff
@@ -1781,3 +1782,15 @@ Support
 |Desktop|Android (Chrome)|iOS (Safari)|
 |:-----:|:--------------:|:----------:|
 |   ✔   |       ✔        |     ✔      |
+
+## Print message \<message>
+
+> \* Print message "Please finish this scenario"
+
+This step adds the specified message in the report and print it in the terminal.
```
Comment (mario-s) [inline docs/STEPS.md:1790]:
```suggestion
This step adds the specified message in the report and prints it in the terminal.
```

**Review by mario-s — APPROVED** [review]

**Comment by nils-wernecke** [comment]
Closes #18

**Pull request merged**